.github/workflows/scripts/push-images.sh

```diff
@@ -11,10 +11,11 @@ fi
 
 echo "Pushing images tagged as $IMAGETAG..."
 
-for img in spire-server spire-agent k8s-workload-registrar oidc-discovery-provider; do
-    gcrimg=gcr.io/spiffe-io/"$img":"${IMAGETAG}"
-    echo "Executing: docker tag $img:latest-local $gcrimg"
-    docker tag "$img":latest-local "$gcrimg"
-    echo "Executing: docker push $gcrimg"
-    docker push "$gcrimg"
+for platform in linux/amd64 linux/arm64/v7 linux/arm64/v8; do
```

Comment:
Will refactor this such the Makefile PLATFORMS env var will be reused. By calling this script from make.

Reply:
nope ! yeah, good catch !